snmp/tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+import time
+import os
+
+from datadog_checks.dev import docker_run
+from datadog_checks.snmp import SnmpCheck
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+COMPOSE_DIR = os.path.join(HERE, 'compose')
+
+
+@pytest.fixture(scope='session', autouse=True)
+def spin_up_snmp():
+    env = {
+        'COMPOSE_DIR': COMPOSE_DIR
+    }
+    with docker_run(
+        os.path.join(COMPOSE_DIR, 'docker-compose.yaml'),
+        env_vars=env
+    ):
+        time.sleep(5)
```

Comment:
Can this be flaky ?

Reply:
it slept in the rake file -- but yes we know the snmp test can be flaky :(

Reply:
Yeah that's what I thought I remembered. If there is no other way to detect if the container is up then I guess we have no choice

Reply:
it logs one line and I don't know if it really shows that it's up. If it's still flaky in this new incarnation, I'll take some time to look into it